### DISCUSSION TITLE
X Plus 3 upgrade

### DISCUSSION THREAD

**Discussion by mohgarr** [community/community#130630] [Other Feature Feedback, Questions, & Ideas]
X Plus 3 upgrade

Question

### Body

Does the connection between the raspberry pi and the main motherboard have to be over the network? I'd prefer not to use a network connection. But either way, that's an excellent guide and write-up. Very good work!

**Comment by ebndev** [comment]
Hi @mohgarr, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? That link is meant for users to provide feedback for Achievements, which leads to GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by mohgarr** [comment]
Yeah, I thought I'd probably sent that to the wrong place. Thanks!
